src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2217,6 +2217,7 @@ private function registerNotifierConfiguration(array $config, ContainerBuilder $
             MobytTransportFactory::class => 'notifier.transport_factory.mobyt',
             SmsapiTransportFactory::class => 'notifier.transport_factory.smsapi',
             EsendexTransportFactory::class => 'notifier.transport_factory.esendex',
+            SendinblueTransportFactory::class => 'notifier.transport_factory.sendinblue',
```

Comment:
This is the mailer factory and not one from the notifier.

Reply:
good catch! done